python/modules/IcePy/Endpoint.cpp

```diff
@@ -134,9 +134,9 @@ namespace IcePy
     PyTypeObject EndpointType = {
         /* The ob_type field must be initialized in the module init function
          * to be portable to Windows without using C++. */
-        PyVarObject_HEAD_INIT(0, 0) "IcePy.Endpoint", /* tp_name */
-        sizeof(EndpointObject),                       /* tp_basicsize */
-        0,                                            /* tp_itemsize */
+        PyVarObject_HEAD_INIT(0, 0) "Ice.Endpoint", /* tp_name */
```

Comment:
need to revert this change. I was testing the direct exposure of this component as part of the Ice module and documenting it in a .pyi file.

Using the autoapi extension (https://sphinx-autoapi.readthedocs.io/en/latest/) seemed to work initially. However, I reverted to the autodoc extension because autoapi does not support our current generated code due to the conditional class definitions.